### ISSUE TITLE
Error realted to a &void type in wrapper generated with `translate wrapper`

### ISSUE THREAD

**Issue by ArtemkaKun** [vlang/c2v#9]
C2V - https://github.com/vlang/c2v/commit/c08add0b0a14157732616c853e2bddc3b0546253
V - 0.2.4 (ccc3271493019feb13ed1c96b035199e159fe863)

.h file - https://github.com/SanderMertens/flecs/blob/master/flecs.h

`fmt error`

```
flecs.v:412:11: error: use `?` instead of `?void`
  410 |     subjects [4]Ecs_entity_t
  411 |     sizes [4]Ecs_size_t
  412 |     ptrs [4]&void
      |              ~~~~
  413 |     match_indices [4]int
  414 |     variables [4]Ecs_var_t

Internal vfmt error while formatting file: flecs.v.
Encountered a total of: 1 errors.
```

**Comment by Wertzui123** [comment]
This is fixed in #14

**Comment by ArtemkaKun** [comment]
Must be tested with this particular case before be closed to make sure it doesn't exist anymore
